src/rgw/rgw_common.h

```diff
@@ -1250,6 +1250,7 @@ struct req_state {
   string zonegroup_endpoint;
   string bucket_instance_id;
   int bucket_instance_shard_id;
+  string redirect_zone_endpoint;
```

Comment:
@yehudasa would be good to change title from [DNM] rgw to 
rgw: add a default redirect field for zones``